src/options/options.html

```diff
@@ -18,21 +18,30 @@
     <div class="setting">
         <div class="setting-title">Email Bundling</div>
         <div class="setting-content">
-            This option is used to bundle labels in the inbox.<br /><br />
+            Bundle emails in your inbox by label<br /><br />
             <input type="radio" name="email-bundling" value="enabled"> Enabled
             <br />
             <input type="radio" name="email-bundling" value="disabled"> Disabled
         </div>
     </div>
     <div class="setting">
-        <div class="setting-title">Email Avatars</div>
+        <div class="setting-title">Colored Avatars</div>
         <div class="setting-content">
-            This option will show a circle with the first letter initial of the sender.<br /><br />
+            Display a colored circle with the first letter initial of the sender<br /><br />
             <input type="radio" name="avatar" value="enabled"> Enabled
             <br />
             <input type="radio" name="avatar" value="disabled"> Disabled
         </div>
     </div>
+    <div class="setting">
+        <div class="setting-title">Profile Photos</div>
```

Comment:
I would actually just have this be part of avatar's options. If avatar is enabled and picture is available, show picture, if picture not available show letter.

Reply:
Oh? That setting was actually specifically inspired by you! Didn't you say something like: "I added this setting to turn avatars off because I just want profile photos anyway"?

Reply:
Found it:

>I also added an option to disable showing the avatars (round circles with letters in them) I only ever found avatars useful when they had the photo of the person in them.

Reply:
@russelldc yes, if profile pictures is working with avatars, I would turn avatars back on!

Reply:
Ahh! :P I thought you didn't want the other avatars at all. Gotcha. I'll roll that back.